html/donate/en.html

```diff
@@ -0,0 +1,98 @@
+</style>
+<script type="text/javascript">
+
+  var _gaq = _gaq || [];
+  _gaq.push(['_setAccount', 'UA-31851927-1']);
+  _gaq.push(['_setDomainName', 'openfoodfacts.org']);
+  _gaq.push(['_trackPageview']);
+
+  (function() {
+    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
+    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
+    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
+  })();
+
+</script>
+
+</head>
+<body>
+
+<div class="hide-for-large-up text-center">
+<img src="https://static.openfoodfacts.org/images/svg/openfoodfacts-logo-en.svg" width="178" height="156" alt="Open Food Facts" />
+</div>
+
+<div class="row" style="width: 100%; margin-left: auto; margin-right: auto; max-width: initial; margin-top:20px;">
+
+<div class="columns large-2 hide-for-medium hide-for-small text-center">
+<img src="https://static.openfoodfacts.org/images/svg/openfoodfacts-logo-en.svg" width="178" height="156" alt="Open Food Facts" />
+</div>
+
+<div class="columns large-5 xlarge-6 xxlarge-7 hide-for-medium hide-for-small">
+
+			<h2 style="margin-bottom:1rem">A message from the Open Food Facts team</h2>
+
+<img src="https://static.openfoodfacts.org/images/svg/crowdsourcing-icon.svg" style="width:240px;float:left;margin-right:5px;" alt="Thank you!" >
+
+<p>Open Food Facts is the "Wikipedia of food", the largest free and open database of food products in the world.</p>
+<p>We started Open Food Facts because we believe information about what we eat is information of public interest. And as a result, it has to be public, free and open.</p>
+<p>Our data is used by citizens to make healthier food choices thanks to our mobile app, by public health researchers to study the impact of food on our health, by journalists for their reporting, and even by the food industry to improve their products. That's the power of food transparency: it benefits everyone!</p>
+<p>When we founded Open Food Facts, we made the clear choice to be totally non-profit to maximize our impact by giving away all our work. We also made the clear choice that we would be totally independent from the industry, and thus to never accept any funding that could put our mission at risk.</p>
+<p>As a result, we started the project on a shoestring, and we still keep our budget tight. From 2012 to 2017 we were all volunteers and our annual budget was less than 1000 euros! The project has grown a lot however, and we now need a bit more to continue our mission.</p>
+<p>We need your help to continue to operate and grow Open Food Facts.Please consider making a donation to protect and sustain Open Food Facts.</p>
+<p>Thank you,</p>
+<p><b>Anca, Florence, Léonore, Sylvia, Charles, Christian, Ludovic, Pierre, Sébastien, Stéphane, Vincent and the whole Open Food Facts team</b></p>
+</div>
+
+<div class="columns large-5 xlarge-4 xxlarge-3 text-center">
+  <center>
```

Comment:
The `center` tag has been [deprecated](https://developer.mozilla.org/de/docs/Web/HTML/Element/center) long ago

Reply:
See #2538